--- a/minissdp.c	2017-04-21 19:24:23.000000000 +0800
+++ b/minissdp.c	2021-06-22 07:01:24.000000000 +0800
@@ -1021,8 +1021,8 @@
 	}
 	if(lan_addr == NULL)
 	{
-		syslog(LOG_WARNING, "SSDP packet sender %s (if_index=%d) not from a LAN, ignoring",
-		       sender_str, source_if);
+		// syslog(LOG_WARNING, "SSDP packet sender %s (if_index=%d) not from a LAN, ignoring",
+		//       sender_str, source_if);
 		return;
 	}
